lib/Runner.js

```diff
@@ -61,30 +66,34 @@ class Runner extends EventEmitter {
           })
           .promise()
           .then(response => {
+            if (timer === 'wall') {
+              const duration = performance.now() - before;
```

Comment:
Why not just `return`?

Reply:
oops, left over from whenN I was doing it differently